source/agora/consensus/EnrollmentManager.d

```diff
@@ -423,6 +423,35 @@ public class EnrollmentManager
         return false;
     }
 
+    /***************************************************************************
+
+        Get the unregistered enrollments in the block
+
+        Params:
+            enrolls = will contain the unregistered enrollments data if found
+
+        Returns:
+            Return true if the unregistered enrollment data was found
+
+    ***************************************************************************/
+
+    public bool getEnrollments (out Enrollment[] enrolls) @trusted
```

Comment:
When you use `out`, memory is not reused. Because `out` will initialize the array to its default value, which is `null`.
In order to reuse memory, you should use `ref`.

Reply:
applied the suggest